pkg/gce-pd-csi-driver/utils.go

```diff
@@ -56,14 +57,30 @@ func NewNodeServiceCapability(cap csi.NodeServiceCapability_RPC_Type) *csi.NodeS
 	}
 }
 
+// Reflect magic below simply clears Secrets map from request.
+func clearSecrets(req interface{}) interface{} {
+	v := reflect.ValueOf(&req).Elem()
+	e := reflect.New(v.Elem().Type()).Elem()
+	e.Set(v.Elem())
+	f := reflect.Indirect(e).FieldByName("Secrets")
+	if f.IsValid() && f.CanSet() && f.Kind() == reflect.Map {
+		f.Set(reflect.MakeMap(f.Type()))
+		v.Set(e)
+	}
+	return req
+}
+
 func logGRPC(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
 	if info.FullMethod == ProbeCSIFullMethod {
 		return handler(ctx, req)
 	}
-	// Note that secrets are not included in any RPC message. In the past protosanitizer and other log
+	// Note that secrets may be included in some RPC messages
+	// (https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/issues/1372),
+	// but the driver ignores them. In the past protosanitizer and other log
```

Comment:
Are you able to do a benchmark test to show that the reflect method does not signficantly increase CPU usage?

Reply:
Hey @msau42 , I added a test to benchmark logGRPC. For 1M iterations, the reflect method increases execution time by only 1%:
```
# Before applying patch with `clearSecrets(req)`
$ go test -c -o /tmp/test ./pkg/gce-pd-csi-driver/ && time /tmp/test -test.run=TestLogGRPC -test.count=1000000
PASS
real	0m20.632s

# After applying patch with `clearSecrets(req)`
$ go test -c -o /tmp/test ./pkg/gce-pd-csi-driver/ && time /tmp/test -test.run=TestLogGRPC -test.count=1000000
PASS
real	0m20.750s
```